docs/README.md

```diff
@@ -900,15 +900,132 @@ where the schema for `CreateProductInput` is
+
+  public entityID(): UUID {
+    return /* the associated entity ID */
+  }
+}
+```
+
+Events and [entities](#4-entities-and-reducers) are closely related because entities represent the application's state after reducing the stream of events. All the events associated with the same entity are reduced together in the order they have occurred. If an event is related to an entity or not, it is domain-dependent. That's why Booster requires you to implement the `entityID` method. Entities and events are somehow coupled, but that coupling is limited to data. Take a look at this event:
+
+```typescript
+@Event
+export class CartPaid {
+  public constructor(
+    readonly cartID: UUID,
+    readonly paymentID: UUID) {}
+
+  public entityID(): UUID {
+    // returning cartID or paymentID depends on the business logic
+    return this.paymentID
+  }
+}
+```
+
+Note:
+
+> An event must be able to find the entity ID value within the arguments. Or well because it got injected directly, or because it is present as a nested attribute. In the case of the `CartPaid` example, the entity ID is injected directly.
+
+In most situations, you can reduce the event stream to an entity like a Cart or Payment. However, in some use cases, the event stream relates to a specific entity without reducing it. For example, a record of sensor values in a weather station; in that case, the station has no particular value to be reduced. You can implement the semantics that best suits your needs.
+
 #### Events naming convention
 
+As with commands, you can name events in any way you want, depending on your application's domain. Though, we recommend you to choose short sentences written in past tense because events are facts that have happened and can't be changed. Some event names would be:
+
+- ProductCreated
+- ProductUpdated
+- ProductDeleted
+- CartItemChanged
+- StockMoved
+
+As with other Booster files, events have their own directory:
+
+```text
+project-root
+├── src
+│   ├── commands
+│   ├── common
+│   ├── config
+│   ├── entities
+│   ├── events <------ They must be here
+│   ├── index.ts
+│   └── read-models
+```
+
 #### Creating events
 
-#### The event store
+The preferred way to create event files is the `new:event` generator, e.g.
+
+```shell
+boost new:event StockMoved --fields productID:string origin:string destination:string quantity:number
+```
+
+That will generate a file called `StockMoved.ts` under the proper `project-root/src/events` directory. You can also create the file manually, but we recommend using the generator and avoid dealing manually with boilerplate code.
+
+Note:
+
+> Running the event generator for an existing EventName, will override the content of the current one. Soon, we will display a warning before overwriting anything. Meanwhile, if you missed a field, just add it to the class because, in Booster, all the infrastructure and data structures are inferred from your code.
```

Comment:
```suggestion
> Running the event generator for an existing EventName, will overwrite the content of the current one. Soon, we will display a warning before overwriting anything. Meanwhile, if you missed a field, just add it to the class because, in Booster there's no hidden magic, all the infrastructure and data structures are inferred from your code.
```

Reply:
Good point @alvaroloes, I'll open a ticket to address it, as the same note is repeated in other parts too.